.github/workflows/testing-dependency.yaml

```diff
@@ -124,5 +94,49 @@ jobs:
           command: poetry run pytest tests/dependency/grpc -s -v
         env:
           PINECONE_API_KEY: ${{ secrets.PINECONE_API_KEY }}
-          INDEX_NAME: ${{ needs.dependency-matrix-grpc-setup.outputs.index_name }}  
-        
\ No newline at end of file
+          INDEX_NAME: ${{ needs.dependency-matrix-setup.outputs.index_name }}  
+
+
+
+
+  dependency-matrix-rest:
+    name: Deps (REST)
+    runs-on: ubuntu-latest
+    needs: dependency-matrix-setup
+    strategy:
+      max-parallel: 5
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.8
+          - 3.9
+          - '3.10'
+          - 3.11
```

Comment:
Everything but `3.8` is commented out in `dependency-matrix-grpc`, is that intentional?

Reply:
Eh, not exactly. Mainly just trying to save some time and resources while iterating. I should bring them back on the grpc job.